GeneralsMD/Code/GameEngine/Source/Common/INI/INIMapCache.cpp

```diff
@@ -152,7 +152,12 @@ void INI::parseMapCacheDefinition( INI* ini )
 	{
 		// maps without localized name tags
 		AsciiString tempdisplayname;
-		tempdisplayname = name.reverseFind('\\') + 1;
+		if(name.reverseFind('\\')) {
+			tempdisplayname = name.reverseFind('\\') + 1;
+		}
+		else if(name.reverseFind('/')) {
+			tempdisplayname = name.reverseFind('/') + 1;
+		}
```

Comment:
It looks like there's cases where this path can have either windows or linux, so to be safe, checking for both.